src/OSFramework/DataGrid/Grid/AbstractGrid.ts

```diff
@@ -271,7 +271,7 @@ namespace OSFramework.DataGrid.Grid {
 
 			columns
 				.map((col) => col.config.binding)
-				.filter((colBinding) => !colBinding.startsWith('$')) // remove Action/Calculated columns
+				.filter((colBinding) => !colBinding.startsWith('$') && colBinding) // remove Action/Calculated columns and the ones that don't have bindings as Group columns
```

Comment:
```suggestion
				.filter((colBinding) => colBinding && !colBinding.startsWith('$')) // remove Action/Calculated columns and the ones that don't have bindings as Group columns
```

Reply:
Only makes sense to see if the column binding starts with the "$", if the column binding has any content.